packages/flutter/lib/src/rendering/view.dart

```diff
@@ -366,6 +366,18 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
     }
   }
 
+  @override
+  void scheduleInitialSemantics() {
+    super.scheduleInitialSemantics();
+    _view.setSemanticsTreeEnabled(true);
+  }
+
+  @override
+  void clearSemantics() {
+    _view.setSemanticsTreeEnabled(false);
```

Comment:
I am unsure about the ordering here. Let's say you have a `View` Widget V1 that is configured with a `FlutterView`. If you now change the widget tree in a frame so that the `View` Widget V1 is replaced with a new `View` Widget V2 configured with the same `FlutterView`, will the FlutterView still have semantics tree enabled? Can you add a test case for this? Something like:

```dart
// [turn on semantics]

FlutterView view = ....
Key key1 = UniqueKey();
Key key2 = UniqueKey();

tester.pumpWidget(
  wrapWithView: false,
  View(
    key: key1,
     view: view,
     child: ...
  ),
);

// expect that setSemanticsTreeEnabled on view is set to true.

tester.pumpWidget(
  wrapWithView: false,
  View(
     key: key2, // make sure this is a new View widget
     view: view,
     child: ...
  ),
);

// expect that setSemanticsTreeEnabled on view is set to true.

```